sentry/src/test/java/io/sentry/GsonSerializerTest.kt

```diff
@@ -543,6 +545,23 @@ class GsonSerializerTest {
                         any<SentryEnvelopeException>())
     }
 
+    @Test
+    fun `empty maps are serialized to null`() {
+        val event = SentryEvent()
+        event.tags = emptyMap()
+        val element = JsonParser().parse(serializeToString(event)).asJsonObject
+        assertNull(element.asJsonObject["tags"])
+    }
+
+    @Test
+    fun `empty lists are serialized to null`() {
+        val transaction = SentryTransaction("tx")
+        val stringWriter = StringWriter()
```

Comment:
can we reuse the method `serializeToString`?

Reply:
Change to serializeToString has been already done in https://github.com/getsentry/sentry-java/pull/1235/. Lets merge #1235 first and then we can refactor the rest of the test methods.

Reply:
makes sense